.circleci/config.yml

```diff
@@ -70,6 +70,32 @@ jobs:
           command: |
             export DIST_ID=$(aws cloudfront list-distributions --query "DistributionList.Items[?Aliases.Items[?@=='<< parameters.bucket >>']].Id | [0]" | tr -d '"')
             aws cloudfront create-invalidation --distribution-id ${DIST_ID} --paths "/*"
+  modified_deploy:
+    docker:
+      - image: cimg/python:3.6
+    parameters:
+      bucket:
+        description: The S3 Bucket to sync files to.
+        type: string
+    steps:
+      - attach_workspace:
+          at: *workspace
+      - aws-s3/sync:
+          from: out
+          overwrite: true
+          to: 's3://<< parameters.bucket >>'
+      - run:
+          name: Rename page HTML files
+          command: ./scripts/modify-deploy-files.sh
+      - aws-s3/copy:
+          arguments: '--content-type "text/html"'
+          from: out/full
+          to: 's3://<< parameters.bucket >>'
```

Comment:
This is a hack to make the routes work as we'd expect. Without this, we'd have to navigate to the *.html file directly rather than the desired route (ex /full.html vs /full). When new pages are added, we'll have to add a new copy step for the page. A better solution should probably be figured out in the future, but this will work for now.